src/isql/isql.h

```diff
@@ -314,23 +314,25 @@ struct IsqlVar
 class IsqlWireStats
 {
 public:
-	IsqlWireStats(Firebird::IAttachment* att);
+	IsqlWireStats(Firebird::IAttachment* att) :
```

Comment:
Maybe we should add `explicit` here?

Reply:
Done, while I consider it as not needed here